planner/core/logical_plan_test.go

```diff
@@ -1792,7 +1792,7 @@ func (s *testPlanSuite) TestUpdateEQCond(c *C) {
 	}{
 		{
 			sql:  "select t1.a from t t1, t t2 where t1.a = t2.a+1",
-			best: "Join{DataScan(t1)->DataScan(t2)->Projection}(test.t.a,Column#25)->Projection",
+			best: "Join{DataScan(t1)->DataScan(t2)->Projection}(test.t.a,Column#25)->Projection->Projection",
```

Comment:
This additional projection is added in join reorder, since the schema of the reordered join contains `t2.a+1`. These 2 projections would be eliminated in the final physical plan, so I think this test change is fine.